.circleci/config.yml

```
@@ -52,6 +52,45 @@ defaults: &defaults
       type: string
       default: release-28.x
 
+aliases:
+  - &setup_dct_env
+    name: Setup DCT environment
+    command: |
+      case "${CIRCLE_BRANCH}" in
+        "master-"*)
+           MINION_IMAGE_VERSION="$(~/project/.circleci/scripts/pom2version.sh ~/project/pom.xml)"
+          ;;
+        "release-"*)
+           MINION_IMAGE_VERSION="release-candidate"
+          ;;
+        "foundation-"20[1-2][0-9])
+           MINION_IMAGE_VERSION="${CIRCLE_BRANCH}"
+          ;;
+        "develop")
+           MINION_IMAGE_VERSION="bleeding"
+          ;;
+        *)
+          MINION_IMAGE_VERSION=b<< pipeline.number >>
```

```suggestion
          MINION_IMAGE_VERSION="${CIRCLE_BRANCH//\//-}"
```

Let's use the same version logic as the Horizon/Sentinel containers - I ran into this in some other branches recently.

---

works for me, I was just leaving things the way they were in 28/develop as much as possible